File: Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Text;

namespace DataGrid
{
    public partial class Order
    {
        public string _id { get; set; } 
        public string userName { get; set; }
        public string productName { get; set; }
        public int quantity { get; set; }
        public string date { get; set; }
    }
}

File: API/API/API/src/models/Order .js
const mongoose = require('mongoose');

const orderSchema = new mongoose.Schema(
    {
        date: {
            type: String,
            required: true,
        },
        userName: {
            type: String,
            required: true,
        },
        productName: {
            type: String,
            required: true,
        },
        quantity: {
            type: Number,
            required: true,
        },
    }
);

const Order = mongoose.model('Order ', orderSchema);

module.exports = Order;
